deploy_contract_01.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#******************************************************************************
+#* Filename:            deploy_contract_01.py
+#* Last Updated:        2019-09-05
+#* Status:              Prototype
+#* Description:         Basic script to assist in contract deployment
+#* Project:             Zilliqa dApp Project
+#******************************************************************************
+#*     Copyright (c) 2019 Well Bred Software Limited (on behalf of DX3 Core)
```

Comment:
Let's create a copyright header that we can all use in the code. Since we use MIT license, it could be as simple as
```
Copyright (c) 2019 Rustiq Technology Ltd & Well Bred Software Ltd
MIT License
```